projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.module.scss';
+import { ButtonGroup, Button, DropdownMenu } from '@wordpress/components';
+
+export const PRODUCT_STATUSES = {
+	ACTIVE: 'active',
+	INACTIVE: 'inactive',
+	ERROR: 'error',
+	ABSENT: 'absent',
+};
+
+const DownIcon = () => (
+	<svg width="15" height="9" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="10 9 4 7">
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="m18.004 10.555-6.005 5.459-6.004-5.459 1.009-1.11 4.995 4.542 4.996-4.542 1.009 1.11Z"
+			fill="#fff"
+		/>
+	</svg>
+);
+
+const ProductCard = ( {
+	productName,
+	productDescription,
+	productIcon,
```

Comment:
We probably can drop the [tautology](https://en.wikipedia.org/wiki/Tautology_(language)) here and just call these props`name`, `description`, `icon`...

Reply:
@oskosk yes, I'll do that.

Reply:
I like the unfriendly tautology idea in general.

🚫 
```es6
const { siteTitle, siteDescription } = getSite();
```

✅ 
```es6
const { title, description } = getSite();
```

Reply:
Done in bb83269195aad5db129f7352997ef8add6958543